src/modules/submissions.js

```diff
@@ -1157,7 +1157,7 @@
 				page_id: mw.config.get( 'wgArticleId' )
 			} ).then( function ( json ) {
 				var triageInfo = json.pagetriagelist.pages[ 0 ];
-				if ( triageInfo && triageInfo.copyvio === mw.config.get( 'wgCurRevisionId' ) ) {
+				if ( triageInfo && triageInfo.copyvio == mw.config.get( 'wgCurRevisionId' ) ) {
```

Comment:
```suggestion
				if ( triageInfo && triageInfo.copyvio === String( mw.config.get( 'wgCurRevisionId' ) ) ) {
```

Might be better to be explicit, as IDEs view use of `==` as suspicious.